### ISSUE TITLE
in the order grid the email is missing

### ISSUE THREAD

**Issue by seansan** [mage-eag/mage-enhanced-admin-grids#91]
grid customized via this extension to show email in column
- we enter new customer, new order via backend
- enter email, products, payment etc
- save order, capture invoice
- the in the order grid the email is missing
- all order from frontend do show email

![image](https://cloud.githubusercontent.com/assets/652395/3004536/54b38d84-dda5-11e3-8663-05ec7f159090.png)

**Comment by Maikel-Koek** [comment]
This column is not available in the 1.0.0 branch.

It's possible after adding the following to the `customgrid.xml`:

```
...
<customer_email model="customgrid/custom_column_order_base" module="customgrid">
    <name>Email</name>
    <group>Customer</group>
    <allow_renderers>1</allow_renderers>
    <config_params>
        <order_field_name>customer_email</order_field_name>
    </config_params>
</customer_email>
...
```

**Comment by mage-eag** [comment]
@Maikel-Koek Just have added it in the 1.0.0 branch, thanks for having pointed this out :)
